migrate/migrate.go

```diff
@@ -21,6 +21,9 @@ import (
 	"github.com/scylladb/gocqlx/qb"
 )
 
+// ErrNoMigrations is returned if the directory provided doesn't contain any migrations, or if a migration file does not contain a CQL statement
+var ErrNoMigrations = errors.New("no migrations were found")
```

Comment:
Why do you thinks this error shall be exported, are there any other errors that should be exported too?

Reply:
Actually no, it is a remnant of #30. It does not need to be exported (at least for now).

Reply:
OK so if it's not exported maybe it should be inlined WDYT?

Reply:
You mean like the rest of the errors like so?
`return fmt.Errorf("no migrations found in %q", dir)`

Sure that could work, we can change it if and when we have to.